src/app/components/transferDetailView/index.tsx

```diff
@@ -64,23 +117,82 @@ function TransferDetailView({
   address,
   hideAddress,
   hideCopyButton,
+  outputScript,
+  outputScriptIndex,
 }: Props) {
+  const { t } = useTranslation('translation', { keyPrefix: 'CONFIRM_TRANSACTION' });
+  const [showScriptOutput, setShowScriptOutput] = useState(false);
+  const styles = useSpring({
+    from: {
+      opacity: 0,
+      y: 24,
+    },
+    to: {
+      y: 0,
+      opacity: 1,
+    },
+    delay: 100,
+  });
```

Comment:
what is the y animation for?
if this is just to show/hide the modal, a CSS transition is preferred. JS animations share resources all other JS scripting, so not ideal for basic UI feedback like this

ref: https://medium.com/neocoast/animations-css-vs-javascript-226903d6976a

Reply:
i noticed that this styling was unnecessary as the component would load up the same way with and without the style added. The current behaviour is smooth as it is. I have removed this now. 
Thanks for sharing the article! Will be mindful of this when adding styling in future